.gitignore

```diff
@@ -19,6 +19,7 @@ temp/babel-plugin-react-intl
 ### Development environments ###
 .idea
 .vscode
+/module.config.js
```

Comment:
Why is this change necessary? @01001110J

Reply:
Is not necessary, it was removed @Squirrel18